jsonargparse_tests/test_core.py

```diff
@@ -167,6 +167,23 @@ def test_default_env(self):
             self.assertFalse(parser.default_env)
 
 
+    def test_env_prefix(self):
+        for env_prefix in [None, True]:
```

Comment:
Don't test for `None`. The test should be such that no change would be needed when in v5.0.0 giving `env_prefix=None` raises an error.

Reply:
Removed